.github/workflows/build-and-test.yml

```diff
@@ -23,24 +23,64 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: sudo apt-get update -y && sudo apt-get install -y libgtk-3-dev libwebkit2gtk-4.0-dev build-essential pkg-config
 
-      - name: Set up Go 1.18
+      - name: Setup Go
```

Comment:
When I looked at this, the tests did not seem to go into the `v2` directory. I tried using `working-directory: ./v2/` but that generated some build constraint errors that I haven't investigated yet.

Reply:
`/wails/v2/cmd/wails/internal/commands/initialise/templates/templates/vue/main.tmpl.go` is a template file, but `//go:embed` in the file will cause an error in the `go test -v ./...` command: 
```
$ go test -v ./...
cmd\wails\internal\commands\initialise\templates\templates\vue\main.tmpl.go:11:12: pattern all:frontend/dist: no matching files found
```

Reply:
We can call it `main.go.tmpl` and it will still work 👍

Reply:
Will be fixed in this PR https://github.com/wailsapp/wails/pull/2169